liblll/Compiler.cpp

```diff
@@ -99,14 +99,28 @@ std::string dev::eth::compileLLLToAsm(std::string const& _src, bool _opt, std::v
 
 string dev::eth::parseLLL(string const& _src)
 {
+	bool failed = false;
 	sp::utree o;
+
 	try
 	{
 		parseTreeLLL(_src, o);
 	}
-	catch (...) {}
+	catch (...)
+	{
+		failed = true;
```

Comment:
The `killBigints(o);` part needs to be executed even in that case to potentially deallocate data.